keyboards/santy65/info.json

```
@@ -0,0 +1,152 @@
+{
+    "manufacturer": "Santy",
+    "keyboard_name": "santy65",
+    "maintainer": "qmk",
+    "bootloader": "qmk-dfu",
```

This is probably more appropriate given this is meant to be built with Elite-C's, as per readme.
```suggestion
    "bootloader": "atmel-dfu",
```